### PR TITLE
Enable compilation on OpenBSD

### PR DESCRIPTION
**Pull request by he4d**
I modified the Makefile so that we can compile rocksdb on OpenBSD.
The instructions for building have been added to INSTALL.md.
The whole compilation process works fine like this on OpenBSD-current

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -674,7 +678,11 @@ coverage:
 	COVERAGEFLAGS="-fprofile-arcs -ftest-coverage" LDFLAGS+="-lgcov" $(MAKE) J=1 all check
 	cd coverage && ./coverage_test.sh
         # Delete intermediate files
-	find . -type f -regex ".*\.\(\(gcda\)\|\(gcno\)\)" -exec rm {} \;
+	ifeq ($(PLATFORM), OS_OPENBSD)
+		gfind . -type f -regex ".*\.\(\(gcda\)\|\(gcno\)\)" -exec rm {} \;
```
Comment (krytarowski) [inline Makefile]: FIND=gfind or FIND=find depending on OS

File: Makefile [hunk 2]
```diff
@@ -821,7 +829,11 @@ CLEAN_FILES += t LOG $(TMPD)
 # regardless of their duration. As with any use of "watch", hit ^C to
 # interrupt.
 watch-log:
-	watch --interval=0 'sort -k7,7nr -k4,4gr LOG|$(quoted_perl_command)'
+	ifeq ($(PLATFORM), OS_OPENBSD)
+		gnuwatch --interval=0 'sort -k7,7nr -k4,4gr LOG|$(quoted_perl_command)'
```
Comment (krytarowski) [inline Makefile]: WATCH=gnuwatch or WATCH=watch

File: Makefile (hunk 1 above)
Comment (he4d) [reply Makefile]: fixed that.. thanks for mentioning

File: Makefile (hunk 2 above)
Comment (he4d) [reply Makefile]: fixed also that

**Comment by he4d** [comment]
Thank you. I need rocksdb for a working IOTA node. With these changes to rocksdb it works flawless on OpenBSD since 3 days 24/7 usage. I will also create a port for rocksdb on OpenBSD.

**Pull request closed without merging**